### DISCUSSION TITLE
Some documentation for MetaField and MetaType properties

### DISCUSSION THREAD

**Discussion by SadiinsoSnowfall** [async-graphql/async-graphql#1077] [General]
Hi,

Could you add some documentation to key types such as registry::MetaField and registry::MetaType ?
Some of their properties change from version to version and specifying what they are used for would make it easier for library users to update their code accordingly.

Sadiinso.

**Comment by sunli829** [comment]
I didn't expect anyone to use them directly 😂, will add documentation later.

**Comment by SadiinsoSnowfall** [comment]
No problem, for the time being I just used the default values / the values used by the lib for its own type and it seems to work 👍
